src/zadatak5/NejednakostranicniTrougao.java

```diff
@@ -0,0 +1,79 @@
+package zadatak5;
+
+/**
+ * 
+ * @author Goran.Kukolj
+ *
+ */
+public class NejednakostranicniTrougao extends Trougao {
+
+	// kreiramo varijable za stranice
+	private double stranicaA;
+	private double stranicaB;
+	private double stranicaC;
+
+	protected NejednakostranicniTrougao() {
+		super();
+	}
+
+	protected NejednakostranicniTrougao(double stranica) {
+		super(stranica);
+	}
+
+	protected NejednakostranicniTrougao(double stranicaA, double stranicaB, double stranicaC) {
+		super();
+		// provera da li moze da se konstruise trougao
+		if (!((stranicaA + stranicaB) > stranicaC && (stranicaA + stranicaC) > stranicaB && stranicaC > stranicaA)) {
+			System.out.println("Zbir bilo koje dve stranice mora biti veci od preostale stranice");
+		} else
+			this.stranicaA = stranicaA;
+		this.stranicaB = stranicaB;
+		this.stranicaC = stranicaC;
+
+	}
+
+	public double getStranicaA() {
+		return stranicaA;
+	}
+
+	public void setStranicaA(double stranicaA) {
+		this.stranicaA = stranicaA;
+	}
+
+	public double getStranicaB() {
+		return stranicaB;
+	}
+
+	public void setStranicaB(double stranicaB) {
+		this.stranicaB = stranicaB;
+	}
+
+	public double getStranicaC() {
+		return stranicaC;
+	}
+
+	public void setStranicaC(double stranicaC) {
+		this.stranicaC = stranicaC;
+	}
+
+	@Override
+	public double izracunajPovrsinu() {
+		double poluobim = (stranicaA + stranicaB + stranicaC) / 2;
```

Comment:
Nemoj da dupliras kod. Izracunaj poluobim na osnovu obima ili obrnuto. Mozda u ovom slucaju to nije nista strasno(i nije) ali kad budes radio na ozbiljnom projektu treba izbegavati pisanje istog koda.